SportsHub/SportsHub.DAL/Repository/CommentRepository.cs

```diff
@@ -17,5 +18,38 @@ public async Task<IEnumerable<Comment>> GetByArticleAsync(int id)
                 .Where(x => x.ArticleId == id)
                 .ToListAsync();
         }
+
+        public async Task AddCommentAsync(Comment comment)
+        {
+            await DbSet.AddAsync(comment);
+        }
+
+        public bool LikeCommentAsync(int commentId)
+        {
+            var comment = GetById(commentId);
+
+            if (comment == null)
+            {
+                return false;
+            }
+
+            comment.Likes++;
```

Comment:
This logic should be on the service layer. Please move it.

Reply:
So no methods at all for this in the repository?

Reply:
Implemented

Reply:
The repository should contain methods that interact with data. No business logic should be in repository.